site/themes/devhouse-theme/layouts/partials/header.html

```diff
@@ -2,5 +2,5 @@
 <h2>devhou.se</h2>
 <ul class="リンク">
     <li><a href="/">ホーム</a></li>
-    <li><a href="/gallery">画像ギャラリー</a></li>
+    <li><a href="/gallery">写真ギャラリー</a></li>
```

Comment:
this one should probably be updated too to match

https://github.com/devhou-se/www-jp/blob/main/python/dailyimage.py#L57

Reply:
Done, missed that one